src/Libraries/DSCPython/CPythonEvaluator.cs

```diff
@@ -146,6 +146,8 @@ public enum EvaluationState { Begin, Success, Failed }
     public static class CPythonEvaluator
     {
         private const string DynamoSkipAttributeName = "__dynamoskipconversion__";
+        private const string DynamoPrintFuncName = "__dynamoprint__";
+        private const string NodeName = "__pythonnodename__";
```

Comment:
Can we name this `__dynamonodename__` instead? That way it is consistent with the other variables we are adding.